recipes/protopuf/all/conanfile.py

```diff
@@ -38,7 +41,7 @@ def source(self):
         get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
 
     def build(self):
-        pass
+        apply_conandata_patches(self)
```

Comment:
```suggestion
```
Could you please move `apply_conandata_patches` to `source()` instead.

The `apply_conandata_patches()` already assumes a structure in the conandata.yml of the patches that is platform independent, so it should be able to be applied in the `source()` method, and only once. In other words, `apply_conandata_patches()` always applies the same patches, so it can be applied in the `source()` safely.